### PR TITLE
Zach-destructibles-base

### PR DESCRIPTION
**Pull request by MarZ4c**
Creates a destructible object that is destroyed when a player reaches a certain size. Attach the player object to the script when creating new a destructible.

### CODE REVIEW

**Comment by farskw0w** [comment]
After reaching 1.5 scale, when the player runs into the white box it disappears. We can't fly around in this branch though, idk why but I'm stuck on the ground and have to jump around. Is it using the right movement script?

**Comment by farskw0w** [comment]
The red box says it's missing a script

File: Assets/Scenes/SampleScene.unity
Comment (CharmedSatyr) [inline Assets/Scenes/SampleScene.unity:1]: There are many changes in this file. Are they all relevant to destructible objects?

**Comment by MarZ4c** [comment]
I'll be getting rid of the red box in this branch was supposed to be used in the hazard branch and just got stuck in this one.

**Review by CharmedSatyr — APPROVED** [review]

**Comment by CharmedSatyr** [comment]
@MarZ4c I resolved your merge conflict.

File: Assets/Scenes/SampleScene.unity
Comment (MarZ4c) [reply Assets/Scenes/SampleScene.unity:1]: I think those changes are the addition of a new object.

**Pull request merged**